README.md

```diff
@@ -244,6 +244,14 @@ The following papers have used code from Uncertainty Baselines:
 
 ## Contributing
 
+### Formatting Code
```

Comment:
High level question: why is this script located under jft/?

Reply:
We depend on code that is in this folder, so we need to be in this folder if we want to import it (it's not a package).

Reply:
For now, we're putting all code that leverages pretraining/finetuning with JFT/I21K models in jft/. I think this makes sense for now as there's a shared utilities required for such pretraining/finetuning.